### PR TITLE
add xsoar to names 

### PR DESCRIPTION
**Pull request by mchasepan**
This is per request from PAN VP Alliances, TPT

## Status
Ready/In Progress/In Hold (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

### CODE REVIEW

File: docs/partners/become-a-tech-partner.md [hunk 1]
```diff
@@ -106,13 +106,13 @@ At a minimum, an integration consists in:
...
+As an automation platform, it shouldn’t surprise you that we’ve also automated our integration testing and deployment. We do nightly tests of each integration in order to confirm functionality and, in order to do this, we request access to your product and/or APIs. Preferably, the instance will have all data related to the integration use cases, so they can be tested. E.g. if one of the use cases is ingesting incidents to Cortex XSOAR, the instance should have demo incidents. This access is not used for any other purpose.
 
-Before submitting the integration, please work with the Palo Alto Networks BD Team to share access
+Before submitting the integration, please work with the Palo Alto Networks BD Team to share access.
```
Comment (ShahafBenYakir) [inline docs/partners/become-a-tech-partner.md:111]: Can we completely remove this part? we. don't enforce or check it. anymore at all.

File: docs/partners/become-a-tech-partner.md [hunk 2]
```diff
@@ -106,13 +106,13 @@ At a minimum, an integration consists in:
...
 
 ## 7. Submit a GitHub Pull Request
 
-Once you have completed development of your content or integration, please submit a *Pull Request* on our GitHub Content Repo. The title of the Pull Request must contain the *Partner ID* we provided earlier.
+Once you have completed development of your content or integration, please [submit a Pull Request](/docs/contributing/contributing) on our GitHub Content Repo. The title of the Pull Request must contain the *Partner ID* we provided earlier.
```
Comment (ShahafBenYakir) [inline docs/partners/become-a-tech-partner.md:115]:
```suggestion
Once you have completed development of your content or integration, please [submit a Pull Request](/docs/contributing/contributing) on our GitHub Content Repo. You will be required to fill in a contribution form that includes details like your *Partner ID* and some other details.
```
Obviously feel free to play with the grammar

**Review by ShahafBenYakir — APPROVED** [review]
See my comments

File: docs/partners/become-a-tech-partner.md (hunk 1 above)
Comment (mchasepan) [reply docs/partners/become-a-tech-partner.md:111]: I went back and forth about it, will remove.

**Pull request merged**